distribution/docs/src/main/resources/content/_sources/registry-store.adoc

```diff
@@ -11,11 +11,6 @@
 
 == {title}
 
-[NOTE]
```

Comment:
@mcalcote, does this section and note relate to https://github.com/codice/ddf/blob/2.13.x/catalog/spatial/registry/registry-api/src/main/java/org/codice/ddf/registry/api/internal/RegistryStore.java? If so, should we consider removing the experimental comment in the code as well and move it to a non-internal package?

Reply:
I think we should be good to remove it from both places :+1:

Reply:
done